### PR TITLE
set default focus on `Restart Device` in SuccessView

### PR DESCRIPTION
**Pull request by 13r0ck**
Sets the focus after a finished install on the `Restart Device` button rather than the show terminal button.

Screenshot from 2021-12-07 00-44-15

### CODE REVIEW

**Review by jackpot51 — APPROVED** [review]

File: src/MainWindow.vala [hunk 1]
```diff
@@ -694,7 +694,7 @@ public class Installer.MainWindow : Gtk.Dialog {
         }
 
         stack.visible_child = encrypt_view;
-        encrypt_view.reset();
+        encrypt_view.reset ();
```
Comment (jacobgkau) [inline src/MainWindow.vala:697]: Not sure if this impacts functionality or not, but having a space here (and on line 756) looks strange and is inconsistent with other places the reset functions are called (e.g. lines 255, 331, and 535 in this file.) Does the change to this line actually do anything?

Comment (13r0ck) [reply src/MainWindow.vala:697]: The space before the parenthesis is actually the correct lint for vala. It is incorrect elsewhere https://github.com/elementary/appcenter/pull/1752#discussion_r766363200

Comment (jacobgkau) [reply src/MainWindow.vala:697]: Looks like it's part of the elementary OS code style, which they'd be following for AppCenter: https://docs.elementary.io/develop/writing-apps/code-style#whitespace 

Fair enough.

**Review by leviport — APPROVED** [review]
Looks good in 21.10. In 20.04, I'm seeing https://github.com/pop-os/installer/issues/256, but that's not related to this PR. The restart button is still focused at the end of the installer in 20.04, so this is still working despite that bug.

**Pull request merged**